### PR TITLE
VTCCA-4180-fix

### PR DESCRIPTION
**Pull request by zahiriqbal**

### CODE REVIEW

File: app/views/createAccount/registrationConfirmation.scala.html [hunk 1]
```diff
@@ -31,11 +31,11 @@
 
 @mainLayout(pageTitle = messages("account.confirmation.title"), showTopNavigation = false) {
 
-
-
-        <div class="govuk-body">
-            <h1 class="govuk-heading-xl">@messages("account.confirmation.heading")</h1>
-            <h2 class="govuk-heading-l">@messages("create.success.personId") @id</h2>
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1>@messages("account.confirmation.heading")</h1>
```
Comment (darrell-plant) [inline app/views/createAccount/registrationConfirmation.scala.html:35]: does the <h1> require a class?

Comment (zahiriqbal) [reply app/views/createAccount/registrationConfirmation.scala.html:35]: @darrell-plant no as it uses outer class styles

Comment (darrell-plant) [reply app/views/createAccount/registrationConfirmation.scala.html:35]: safer to make it explicit - that is what plat-ui has advised

**Comment by darrell-plant** [comment]
LGTDz 👍

**Comment by habeeb01** [comment]
LGTM 👍

**Pull request merged**